server/datastore/mysql/activities_test.go

```diff
@@ -86,4 +98,12 @@ func TestNewActivity(t *testing.T) {
 	assert.Len(t, activities, 1)
 	assert.Equal(t, "fullname", activities[0].ActorFullName)
 	assert.Equal(t, "test2", activities[0].Type)
+
+	opt = fleet.ListOptions{
```

Comment:
I added this to test the cleanup between tests (truncate tables) that even with a PerPage of 10, it would return only the 2 created in this test.